test/integration/MRC20.spec.ts

```diff
@@ -59,7 +59,10 @@ describe('Token wrapper tests', () => {
     const amount = 123_000_000n
     let operation = await usdcContract.increaseAllowance(
       'AU1wN8rn4SkwYSTDF3dHFY4U28KtsqKL1NnEjDZhHnHEy6cEQm53',
-      amount
+      amount,
+      {
+        coins: Mas.fromString('0.1'),
+      }
```

Comment:
i think u needed it fot the first execution with your wallet( but you could probably delete it now.) it will avoid to spend too much coins in the CI wallet